tests/data/eurocrops/data.py

```diff
@@ -12,15 +12,19 @@
 from rasterio.crs import CRS
 from shapely.geometry import Polygon, mapping
 
-SIZE = 100
+SIZE = 1000
```

Comment:
128 should be sufficient if we go with an 8x8 grid

Reply:
It is in projection units, 100 would be too small since it would by 10x10.

Reply:
I added a comment about it.